### PR TITLE
BRS-374 Connecting homepage to CMS

### PR DESCRIPTION
**Pull request by DayKirby-OXD**
### Jira Ticket:
BRS-123

### Jira Ticket URL:
https://bcparksdigital.atlassian.net/browse/BRS-123

### Description:
Connect content for new homepage design to CMS
Updated components to expand functionality of LinkCard components
Updated bg image to use gastby optimizing, also adjusted size re Harry's updated design
Removed MUI from homepage, other styling cleanup
Structure of home page for test/prod etc for testing + launch can be found in ticket comments
NB: had to adjust graphql query for all page templates within "homepage" object to not conflict with new page component structure ("HTML" field in homepage query is removed in several files) - these aren't being used; the graphql queries on all templates pages should be  revisited eventually

### CODE REVIEW

**Review by cole — CHANGES_REQUESTED** [review]
Looks good overall, I noted a few small things that need fixing.

I think we'll need some direction in the ticket on how to update the home page content, and also to note that all of the landing page cards will need to be updated to specify the variation.

File: src/cms/components/parks/page-section.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "info": {
     "name": "PageSection",
     "icon": "book-open",
+
```
[the remark this answers is not in the document]
Comment (DayKirby-OXD) [reply src/cms/components/parks/page-section.json:6]: unintentional, fixed

File: src/staging/src/pages/index.js [hunk 2]
```diff
@@ -1,26 +1,14 @@
...
 import Header from "../components/header"
 import Footer from "../components/footer"
-import Zone from "../components/zone"
 import MainSearch from "../components/search/mainSearch"
-import { Container } from "@material-ui/core"
-import { useMediaQuery } from "react-responsive"
-
-// Temporary in-code images
-import imgSearchBg from "../images/home/image006.png"
-import imgCampingInfo from "../images/home/image012.png"
```
[the remark this answers is not in the document]
Comment (DayKirby-OXD) [reply src/staging/src/pages/index.js:12]: done

File: src/staging/src/pages/index.js [hunk 3]
```diff
@@ -67,243 +57,41 @@ export const query = graphql`
...
-                      </h5>
-                      <div className="card-button">
-                      Learn more <i className="fa fa-chevron-circle-right"></i>
-                      </div>
-                    </div>
-                  </a>
-                </div>
-              </div>
+          <div className="park-search-carousel">
+            <StaticImage src="../images/home/image006.png"
```
[the remark this answers is not in the document]
Comment (DayKirby-OXD) [reply src/staging/src/pages/index.js:74]: i'll ask harry for a bigger image. if he doesn't have one we might have to revisit.

File: src/staging/src/styles/home.scss [hunk 4]
```diff
@@ -11,6 +11,10 @@ $tabletBreakpoint: 768px;
   overflow-x: hidden;
 }
 
+#home {
```
[the remark this answers is not in the document]
Comment (DayKirby-OXD) [reply src/staging/src/styles/home.scss:14]: done

**Comment by DayKirby-OXD** [comment]
> Looks good overall, I noted a few small things that need fixing.
> 
> I think we'll need some direction in the ticket on how to update the home page content, and also to note that all of the landing page cards will need to be updated to specify the variation.

Content plus component structure is in ticket comments, made this clearer in ticket

File: src/staging/src/pages/index.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (DayKirby-OXD) [reply src/staging/src/pages/index.js:74]: Checked the design, it's supposed to auto-margin if wider than 1440, so adjusting the layout.

**Review by cole — APPROVED** [review]

File: src/staging/src/pages/index.js [hunk 5]
```diff
@@ -1,26 +1,14 @@
 import React from "react"
 import { graphql } from "gatsby"
+import { StaticImage } from "gatsby-plugin-image"
+import { useMediaQuery } from "react-responsive"
```
Comment (BcGovNeal) [inline src/staging/src/pages/index.js:4]: No longer used, please remove

File: src/staging/src/components/pageContent/pageContent.js [hunk 6]
```diff
@@ -22,12 +23,32 @@ export default function PageContent({ contentType, content }) {
...
+          variation={content.variation}
+          buttonText={content.buttonText}
         ></LinkCard>
       )
     }
+  if (contentType === "parks.card-set") {
+    return (
+      <div className="row">
+        {content.cards.map(card =>
+          <LinkCard
```
Comment (BcGovNeal) [inline src/staging/src/components/pageContent/pageContent.js:39]: Missing `key` prop on the list of `LinkCard` components.  You should see the "missing key prop" errors in your console

File: src/staging/src/pages/index.js [hunk 7]
```diff
@@ -67,243 +57,41 @@ export const query = graphql`
...
+              <PageContent contentType={content.strapi_component} content={content}></PageContent>
             </div>
-          </Container>
-        </>
-      )}
-      <Container className="max-width-override" fixed disableGutters>
+          )}
+        </div>
+      </div>
+      <div className="home-max-width-override" fixed disableGutters>
```
Comment (BcGovNeal) [inline src/staging/src/pages/index.js:90]: Can remove `fixed disableGutters` here.  They were here when the `div` was a `Container` component from Material UI.  They don't do anything now and are creating errors in the console

**Review by BcGovNeal — CHANGES_REQUESTED** [review]
Overall it looks great!  The card set component works great.

Just a few things needing clean up

File: src/staging/src/pages/index.js (hunk 5 above)
Comment (DayKirby-OXD) [reply src/staging/src/pages/index.js:4]: whoops you're right will do

File: src/staging/src/components/pageContent/pageContent.js (hunk 6 above)
Comment (DayKirby-OXD) [reply src/staging/src/components/pageContent/pageContent.js:39]: indeed, thx.

File: src/staging/src/pages/index.js (hunk 7 above)
Comment (DayKirby-OXD) [reply src/staging/src/pages/index.js:90]: yep, thx.

**Review by BcGovNeal — APPROVED** [review]

**Pull request merged**